src/components/ArticleById.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "./Api";
+
+const ArticleById = () => {
+
+  const {article_id} = useParams()
+  const [article, setArticle] = useState([])
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    getArticleById(article_id).then((response) => {
+        setArticle(response.article)
+        setLoading(false)
+    })
+  }, [article_id])
+
+  if(loading === true){
+    return <p>Loading...</p>
+  }
+  return (
+    <section id="articleById" key={article.article_id}>
+    <br />
+      <h4 id="articleTitle"><strong>{article.title}</strong></h4>
+      <dl>
+        <dt id="authorTitle"><strong>Author:</strong></dt>
+        <dd>{article.author}</dd>
+        <dt id="catergoryTitle"><strong>Category:</strong></dt>
+        <dd>{article.topic}</dd>
+        <dt id="bodyTitle"><strong>Article:</strong></dt>
+        <dd>{article.body}</dd>
```

Comment:
Because the body is the main focus of this page I might suggest separating it out from these details and have it be on its own, its still very much a card structure